spec/e2e/features/listings/listing.feature

```diff
@@ -1,24 +1,26 @@
 Feature: Listings
-    As a web user
-    I should be able to view a listing
+  As a web user
+  I should be able to view a listing
 
-    Scenario: Attempting to go to a listing page using an invalid ID
-      Given I try to go to a listing page with an invalid ID
-      Then I should be redirected to the welcome page
+  Scenario: Attempting to go to a listing page using an invalid ID
+    Given I try to go to a listing page with an invalid ID
+    #give time for the lookup failure and redirect
+    And I wait "5" seconds
```

Comment:
do we have any control over the lookup request, like somewhere we could do some kind of `await` on a response?

Reply:
great question. we can't wait for the ajax request, but we can add a wait for the success condition. will try this another way.